index.js

```diff
@@ -358,9 +352,7 @@ async function write (opts, pkg, { log } = {}) {
   const pkgPath = path.resolve(opts.cwd, 'package.json');
   // Write package json
   log.info(`Writing package.json\n${pkgPath}`);
-  await fs.outputJSON(pkgPath, pkg, {
-    spaces: opts.spacer || 2
```

Comment:
I am guessing we can drop the spacer option if we decide this is a breaking change? Or maybe for conversion sake we want to leave it in case folks want to run this on an existing file and force an indent?

Reply:
ok yeah lets call this resolved and leave it.